routes/api/v1/playlists.js

```diff
@@ -4,7 +4,17 @@ var router = express.Router();
 const environment = process.env.NODE_ENV || 'development';
 const configuration = require('../../../knexfile')[environment];
 const database = require('knex')(configuration);
-
+router.get('/', (request, response) => {
+  database('playlists')
+    .select("id", "title", "created_at as createdAt", "updated_at as updatedAt")
+    .then(result => {
+      response.status(200).send({data: result})
+    })
+    .catch(error => {
+      console.log(error)
+      response.status(500).json({ error: "Oops, something went wrong!" });
```

Comment:
I like this, and I think based on the feedback we received yesterday we should change all our 500 errors to this so we don't send the error through to the user. I'll make a chore for that real quick.